benchmarks/benchmark_serving.py

```diff
@@ -117,7 +117,7 @@ def sample_sharegpt_requests(
         prompt_len = len(prompt_token_ids)
         output_len = len(completion_token_ids
                          ) if fixed_output_len is None else fixed_output_len
-        if prompt_len < 4 or output_len < 4:
+        if prompt_len < 4 or (fixed_output_len is not None and output_len < 4):
```

Comment:
Can you make this change consistent with the one you made in `sample_hf_requests`?